demos/Dockerfile

```diff
@@ -13,10 +13,16 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install intl \
     && docker-php-ext-install pdo pdo_mysql
 
+#Update node to v12
+RUN curl -sL https://deb.nodesource.com/setup_12.x  | bash -
+RUN apt-get -y install nodejs
+
 WORKDIR /var/www/html/
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 
+# update npm - apt-get is not installing the latest version.
+RUN npm install -g npm
```

Comment:
should be put probably after install nodejs

Reply:
As long as it is before doing any installation. Besides, I have tested it and it was ok.